Information/CenterOfMass.FCMacro

```diff
@@ -596,13 +601,13 @@ class CenterofmassWidget(QtGui.QWidget):
         tu = FreeCAD.Units.parseQuantity
         if defDensity<self.defaultDensitySpin.maximum() and defDensity>self.defaultDensitySpin.minimum():
             self.material_database['default'] = float(tu('{}'.format(defDensity)+self.comboUnit.currentText()) / tu('kg/m^3'))
-    
+
     def setalltodefaultdensity_stateChanged(self, state):
         for sol in range(self.solid_count):
             self.materials[sol].combo.setCurrentIndex(0)
-            self.materials[sol].combo.setCurrentText('default');                    
+            self.materials[sol].combo.setCurrentText('default');
```

Comment:
Optional: extra semi-colon.

Reply:
@galou like:   
`self.materials[sol].combo.setCurrentText('default');;`  
Or are you referring to line 602 ?

Reply:
I thought `self.materials[sol].combo.setCurrentText('default')`, i.e. without semi-colon.

Reply:
Done in 04011ae